docs/en/users/05_Configuration.md

```diff
@@ -171,9 +171,9 @@ What is meant by "CSS path of articles on the original site" actually correspond
 
 #### Example: Rue89
 
-To find this path, you must go to the address of one of the truncated articles (for example http://www.rue89.com/2013/10/15/prof-maths-jai-atteint-lextase-dihn-pedagogie-inversee-246635). You must then look for the "block" of HTML corresponding to the content of the article (in the source code!).
+To find this path, you have to go to the address of one of the truncated articles (for example, http://www.rue89.com/2013/10/15/prof-maths-jai-atteint-lextase-dihn-pedagogie-inversee-246635). You look have to look for the "block" of HTML that corresponds to article content (in the source code!).
```

Comment:
Is this "CSS path" concept is intended to make it more comprehensible @marienfressinaud? I'm not sure if we shouldn't just call it a CSS selector; it's difficult(ish) either way.

Reply:
Oops, I forgot your comment! Yes it was intended to make it more comprehensible but I'm really not sure it is. I think the correct word is "xpath", but it's even less comprehensible… "CSS selector" might be the best choice for the moment, but we should be consistent with the interface. So for the moment I suggest to stay with "CSS path", and change it in another PR later if we think it's important :)

Reply:
Btw, the French translation that we use is "Sélecteur CSS", which literally means "CSS selector" ^^